bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,333 @@
+        """Get all available genres."""
+        await ctx.send(f"Currently available genres: {', '.join(f'`{genre}`' for genre in self.genres.keys())}")
+
+    @games.command(name='search', aliases=['s'])
+    async def search(self, ctx: Context, *, search: str) -> None:
+        """Find games by name."""
+        lines = await self.search_games(self.http_session, search)
+
+        await LinePaginator.paginate((line for line in lines), ctx, Embed(title=f'Game Search Results: {search}'))
+
+    @games.command(name='company', aliases=['companies'])
+    async def company(self, ctx: Context, amount: int = 5) -> None:
+        """
+        Get random Game Companies companies from IGDB API.
+
+        Support amount parameter. Max is 25, min is 1.
+        """
+        if amount > 25:
+            await ctx.send("You can't get more than 25 companies at once.")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 company.")
+            return
+
+        companies = await self.get_companies_list(self.http_session, amount, random.randint(0, 150))
+        pages = await self.get_company_pages(companies)
+
+        await ImagePaginator.paginate(pages, ctx, Embed(title='Random Game Companies'))
+
+    async def get_games_list(self,
+                             client: ClientSession,
+                             limit: int,
+                             genre: str = None,
+                             sort: str = None,
+                             additional_body: str = "",
+                             offset: int = 0) \
+            -> List[Dict[str, Any]]:
+        """Get Games List from IGDB API."""
+        # Create body of IGDB API request, define fields, sorting, offset, limit and genre
+        body = f"""fields cover.image_id, first_release_date, total_rating, name, storyline, url, platforms.name,
+status, involved_companies.company.name, summary, age_ratings.category, age_ratings.rating, total_rating_count;
+{f'sort {sort};' if sort else ''}
+offset {offset};
+limit {limit};
+{f'where genres = ({genre});' if genre else ''}
+{additional_body}"""
+
+        # Do request to IGDB API, create headers, URL, define body, return result
+        async with client.get(
+                url=BASE_URL + "games",
+                data=body,
+                headers=HEADERS
+        ) as resp:
+            return await resp.json()
+
+    async def get_pages(self, data: List[Dict[str, Any]]) -> List[Tuple[str, str]]:
+        """Generate all game pages, do additional requests to IGDB API."""
+        pages = []
+        for game in data:
+            pages.append(await self.create_page(game))
+        return pages
+
+    async def create_page(self, data: Dict[str, Any]) -> Tuple[str, str]:
+        """Create content of Game Page."""
+        # Create page content variable, what will be returned
+        page = ""
+
+        # Get keys to one variable. Used for code style
+        keys = data.keys()
```

Comment:
Why is this needed?  The use of it seems inconsistent with for instance L236 and L224. Might as well use data in those places?